controllers/topics/index.js

```diff
@@ -172,6 +173,7 @@ const followTopicV2 = async (req, res) => {
       secondDetailUser = await UsersFunction.findUserById(User, secondDetailUserId);
     }
 
+    let prevUserToken = await createToken(secondDetailUser.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The creation of `prevUserToken` using `createToken` function seems fine, but there's no null check for `secondDetailUser`. If `secondDetailUser` is null or undefined, this will throw an error.
```diff
+    let prevUserToken = await createToken(secondDetailUser.user_id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->